### ISSUE TITLE
Failed to delete one APIBInding tenancy.kcp.dev object

### ISSUE THREAD

**Issue by wangke19** [kcp-dev/kcp#1755]
** Describe the bug **
When delete one APIBInding tenancy.kcp.dev object, it was stuck in the process of deletion.

```
$ k version -oyaml
clientVersion:
  buildDate: "2022-06-15T14:22:29Z"
  compiler: gc
  gitCommit: f66044f4361b9f1f96f0053dd46cb7dce5e990a8
  gitTreeState: clean
  gitVersion: v1.24.2
  goVersion: go1.18.3
  major: "1"
  minor: "24"
  platform: darwin/amd64
kustomizeVersion: v4.5.4
serverVersion:
  buildDate: "2022-08-09T18:22:25Z"
  compiler: gc
  gitCommit: c4c1214
  gitTreeState: dirty
  gitVersion: v1.24.3+kcp-v0.7.1
  goVersion: go1.17.13
  major: "1"
  minor: "24"
  platform: linux/amd64
```

** To Reproduce **
1. Create one personal workspace.2. 
```
$ kubectl kcp workspace create  kews1 --enter
Workspace "kews1" (type root:universal) created. Waiting for it to be ready...
Workspace "kews1" (type root:universal) is ready to use.
Current workspace is "root:rh-sso-xxx:kews1" (type "root:universal").
```

3. Check all APIBIndings 
```
$ k  get APIBinding
NAME                  AGE
apiresource.kcp.dev   15m
scheduling.kcp.dev    15m
tenancy.kcp.dev       15m
workload.kcp.dev      15m
```

4. Delete all apibindings 
```
$ k delete apibinding apiresource.kcp.dev
apibinding.apis.kcp.dev "apiresource.kcp.dev" deleted

$ k delete apibinding scheduling.kcp.dev
apibinding.apis.kcp.dev "scheduling.kcp.dev" deleted

$ k delete apibinding workload.kcp.dev
apibinding.apis.kcp.dev "workload.kcp.dev" deleted

$ k delete apibinding tenancy.kcp.dev  # It hangs, the object wasn’t deleted, have to ctrl +c to stop.
apibinding.apis.kcp.dev "tenancy.kcp.dev" deleted
```

Checked the yaml file, deletion status as below,
```
$ k get  apibinding/tenancy.kcp.dev -o yaml
…
- lastTransitionTime: "2022-08-16T02:43:53Z"
    message: 'Get "https://kcp-stable.apps.kcp-xxxx.com/services/workspaces/root:rh-sso-xxx:kews1/all/apis/tenancy.kcp.dev/v1beta1/workspaces":
      x509: certificate is valid for *.apps.xxx.com,
      api.kcp-xxxx.com, rh-api.kcp-xxxx.com,
      not apiserver-loopback-client'
    reason: ResourceDeletionFailed
    severity: Error
    status: "False"
    type: BindingResourceDeleteSuccess
```

** Expected Results **
Apibinding object tenancy.kcp.dev can be removed.

**Comment by kasturinarra** [comment]
@wangke19 just a quick check, this might be working, but how do we know this has already been fixed ? Did we have any other PR tracking this work ? Also as per QE process please help add /label qe-approved and then add the detailed steps that you used for verification, thanks !!

**Comment by wangke19** [comment]
@kasturinarra Okay, thank you for your suggestion. I tried to searche for related errors, but couldn't find a PR to confirm if it was fixed. I will re-open the issue. 
@ncdc Could you help to confirm which PR has fixed this issue? Thanks.

**Comment by ncdc** [comment]
This was fixed by #1860 
/close
